src/zenml/utils/logging_utils.py

```diff
@@ -360,12 +377,14 @@ def setup_run_logging(
     """
     log_metadata = get_run_log_metadata(pipeline_run)
     log_metadata.update(dict(source=source))
+    name = f"zenml.pipeline_run.{pipeline_run.id}.{source}"
```

Comment:
We briefly talked about this before, but currently, there is a unique check with an entity id and its `source`. That means, that I can not have two log responses within our DB that belong to the same pipeline run with the same source such as `client`. But if we ever change this behaviour, this name might become a problem. As far as I can see, you are using this as a key while registering emitters. How about we switch the source with the `log_model.id`?